update_ui_tests.sh

```diff
@@ -0,0 +1,23 @@
+export PROJECT_UI_TESTS_PATH=/home/circleci/project/planet4-uitests/
```

Comment:
Not sure of this path 🤔

Reply:
It seems like you are in the correct path. 
From the circleCI documentation: 
https://circleci.com/docs/2.0/configuration-reference/#job_name
```
In which directory to run the steps. Default: ~/project (where project is a literal string, not the name of your specific project). 
```
Since in line 13 you are cloning it where you are (Without having modified the default in the circleCI config), it will create a subsdirectory planet4-uitests (which is the one you are mentioning above).